src/release.sh

```diff
@@ -11,58 +11,63 @@ fi
 cd "$(dirname "$0")/.."
 mkdir -p "$ver"
 
-version=$(curl -sS https://api.github.com/repos/openshift/origin/releases \
+api_user=""
+if [ -n "${GITHUB_API_USER}" ]; then
+    api_user="${GITHUB_API_USER}@"
+fi
+
+version=$(curl -sS https://${api_user}api.github.com/repos/openshift/origin/releases \
   | jq --arg ver "$ver" --raw-output \
       '.[]| select((.prerelease|not) and (.tag_name|startswith($ver))) | .tag_name' \
   | head -n 1)
 
-helm_version=$(curl -sS https://api.github.com/repos/helm/helm/releases \
-  | jq --raw-output \
-      '.[]| select(.prerelease|not) | .tag_name' \
-  | sed '/v3\./d' \
-  | head -n 1)
+helm_version_2=v2.16.4
+helm_version_3=v3.1.2
```

Comment:
Why do you hard-code the version number here? Is this technically required?

Reply:
Well, the previous commands do not work anymore, because there is still development happening for v2, new releases come up for both v2 and v3 (https://github.com/helm/helm/releases). So, some simple `jq` and `head` do not work anymore. I'm happy for suggestions for distinguishing the releases. For now, I hardcoded it, yes.

> As for the variable name, I'd suggest to use helm2_version and helm3_version. Parsing the two different versions is simply a matter of changing the value in the sed command above (v3 ignores version 3, v2 would ignore version 2 and yield the latest version 3).

I don't get that...